packages/vite/src/module-runner/evaluatedModules.ts

```diff
@@ -135,10 +135,9 @@ const prefixedBuiltins = new Set([
 // transform file url to id
 // virtual:custom -> virtual:custom
 // \0custom -> \0custom
-// /root/id -> /id
-// /root/id.js -> /id.js
```

Comment:
Was this one removed because it didn't happen? If it still happens, I think it would be helpful if this line is kept.

Reply:
Ah, makes sense.